e2e/tests/functional/plugins/lad/lad.e2e.spec.js

```diff
@@ -0,0 +1,131 @@
+
+        await subscribeToTelemetry(page, sineWaveObject.uuid);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+    test('telemetry value exactly matches latest telemetry value received in fixed time', async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
+        const { myItemsFolderName } = openmctConfig;
+
+        await page.locator('button.c-mode-button').click();
+        await page.locator('[data-testid="conductor-modeOption-realtime"]').click();
+
+        const sineWaveObject = await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Test Sine Wave Generator"
+        });
+
+        await page.locator(`text=${myItemsFolderName}`).first().click();
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'LAD Table',
+            name: "Test LAD Table"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.c-lad-table-wrapper');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        let getTelemValuePromise = new Promise(resolve => page.exposeFunction('getTelemValue', resolve));
+
+        await subscribeToTelemetry(page, sineWaveObject.uuid);
+        await changeToFixedTime(page);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+});
+
+/**
+ * Util for subscribing to a telemetry object by object identifier
+ * @param {import('@playwright/test').Page} page
+ * @param {string} objectIdentifier identifier for object
+ */
+async function subscribeToTelemetry(page, objectIdentifier) {
+    await page.evaluate(async (telemetryIdentifier) => {
+        const telemetryObject = await window.openmct.objects.get(telemetryIdentifier);
+        const metadata = window.openmct.telemetry.getMetadata(telemetryObject);
+        const formats = await window.openmct.telemetry.getFormatMap(metadata);
+        window.openmct.telemetry.subscribe(telemetryObject, (obj) => {
+            const sinVal = obj.sin;
+            const formattedSinVal = formats.sin.format(sinVal);
+            window.getTelemValue(formattedSinVal);
+        });
+    }, objectIdentifier);
+}
+
+/**
+ * This function uses the time conductor to change to a fixed time of 1 minute
```

Comment:
nice

Reply:
We have these utility functions already in [our Time Conductor e2e tests](https://github.com/nasa/openmct/blob/f1c85933c3575cde42267180bfd34a2a44ba3bcf/e2e/tests/functional/plugins/timeConductor/timeConductor.e2e.spec.js#L224). I think we should extract these as appActions and use those

Reply:
Try to move this into a shared app Action